pygeodes.utils.exceptions.DataAssetMissingException
===================================================

.. currentmodule:: pygeodes.utils.exceptions

.. autoexception:: DataAssetMissingException